docs/devops/gcp/README.md

```diff
@@ -42,11 +42,16 @@ gcloud services enable run.googleapis.com
 gcloud services enable iam.googleapis.com
 gcloud services enable secretmanager.googleapis.com
 gcloud services enable artifactregistry.googleapis.com
+gcloud services enable cloudresourcemanager.googleapis.com
 ```
 
 ## Provision infrastruture
 
-From `infra/terraform/gcp` folder run these commands from your terminal:
+To allow Terraform to provision services on GCP we need to give it a bucket to store it's state.
+
+In GCP web console navigate to `Cloud Storage` and create a new bucket with the `Create Bucket` option. Set a unique name for the bucket and adjust `infra/gcp/terraform/main.tf`.
```

Comment:
I assume this is not something that can be automated, right? If unsure, you may want to check with @dgonzalez

Reply:
The state bucket is one the things that need to be pre-configured manually before TF can run.